src/javaRosa.js

```diff
@@ -1393,6 +1393,9 @@ define([
             } else {
                 _this.__callOld();
             }
+            if (path.contains('casedb')) {
+                window.analytics.usage("Case Management", "Form Builder", "Click and Drag");
```

Comment:
The general case should go here: https://github.com/dimagi/Vellum/blob/master/src/core.js#L757-L764

For output values it should go https://github.com/dimagi/Vellum/pull/625/files/cf5fbdbf288457b26a3cda2c3b33cb798eb90922#diff-e5b3eca8407b45b132e0147b8392192cR1390

Vellums uses a plugin structure and this function is overriding the one in core (and calls the old one (`_this.__callOld()`) if it's not an output value)

It also needs the `if (window.analytics)` wrapper so it doesn't fail in standalone. Not sure why we haven't made a helper for that yet